servant/src/Servant/API/ContentTypes.hs

```diff
@@ -65,7 +65,6 @@ module Servant.API.ContentTypes
     , AllMime(..)
     , AllMimeRender(..)
     , AllMimeUnrender(..)
-    , eitherDecodeLenient
```

Comment:
From the comments below it seems that with currently supported aeson versions `eitherDecodeLenient` and `eitherDecode` have the same behavior. 

What about making `eitherDecodeLenient` an alias of `eitherDecode` and just mark it as deprecated? That way it would not break the API.